main/src/library/Debug.flix

```diff
@@ -11,6 +32,24 @@ mod Debug {
     import java.util.Arrays
     import java.util.Objects
 
+    ///
+    /// Prints `x` to the standard out.
+    ///
+    pub def dprint(x: a): Unit \ Debug with ToString[a] =
+        unchecked_cast({
+            let s: String = ToString.toString(x);
+            System.out.print(s)
+        } as _ \ Debug)
+
+    ///
+    /// Prints `x` to the standard out followed by a new line.
+    ///
+    pub def dprintln(x: a): Unit \ Debug with ToString[a] =
+        unchecked_cast({
+            let s: String = ToString.toString(x);
+            System.out.println(s)
+        } as _ \ Debug)
+
```

Comment:
These should print to stderr, not stdout.

Rust agrees: https://doc.rust-lang.org/std/macro.dbg.html

Reply:
https://github.com/flix/flix/issues/11618